release-calendar/src/client/components/App.tsx

```diff
@@ -15,39 +15,106 @@
  */
 import * as React from 'react';
 import {Calendar} from './Calendar';
+import {Cell, Grid, Row} from '@material/react-layout-grid';
+import {ChannelTable} from './ChannelTable';
 //TODO: remove DATA and instead populate with data from ./test/development-data.ts
+import '@material/react-card/index.scss';
+import '@material/react-layout-grid/index.scss';
 import {DATA} from '../models/data';
 import {EventSourceInput} from '@fullcalendar/core/structs/event-source';
 import {Header} from './Header';
+import {getChannelRTVs} from '../models/release-channel';
 import {getEvents} from '../models/release-event';
+import Card, {
+  CardActionIcons,
+  CardActions,
+  CardPrimaryContent,
+} from '@material/react-card';
 
 interface AppState {
   events: EventSourceInput[];
+  currentRTVs: string[];
+  selectedChannel: number;
```

Comment:
What does this number represent? The index? Ideally it would be a Channel enum type, not number.

Reply:
on line 61 of ChannelTable, a list component expects and only accepts a number in as a parameter. `selectedIndex={this.props.selectedChannel}`It would be very tricky to have it as the enum and I would have to convert it to a number in order to pass it here anyway, but I can probably still do it.... is it worth it?